markdown/3.x/en/api/ff-asn.api.md

```diff
@@ -94,5 +94,7 @@ ___
 ### Properties
 | Name | Description |
 | ---- | ----------- |
-| **submit-on-input**&nbsp;(String) **Options**&nbsp;"true",&nbsp;"false" (default: "false") | If this attribute is set to true, the input fields accept values. If set to false, you cannot focus the input fields. |
+| **disable-input-fields**&nbsp;(String) **Options**:&nbsp;"true",&nbsp;"false" (default: "false") | If this attribute is set to true, the input fields accept values. If set to false, you cannot focus the input fields. |
+| **submit-on-input**&nbsp;(String) **Options**:&nbsp;"true",&nbsp;"false" (default: "false") | If this is set to true a filter request is made immediately upon key stroke instead of waiting for ENTER key press. |
```

Comment:
```suggestion
| **submit-on-input**&nbsp;(String) **Options**:&nbsp;"true",&nbsp;"false" (default: "false") | If this is set to true, a filter request is made immediately upon key stroke instead of waiting for an ENTER key press. |
```

Reply:
I agree with adding the comma. However, how many ENTER keys do you have on your keyboard? OK, I know there can be 2 when you have (the) numeric keyboard, but usually we know which ENTER we are talking about, so it is `the ENTER`. I would skip the article, but some googling tells me `the ENTER` is OK.

Reply:
The `an` goes with `press`. It is a pressing of the "ENTER key".
There can be many presses, therefore `the` is not suitable.
Compare: `instead of waiting for a key press`. Now exchange `key` for the more specific `ENTER key`. It remains `instead of waiting for an ENTER key press`.

Reply:
Now I see what you meant. I must have missed this "press" word. Yes, now I can agree. I thought you referred to `ENTER key`. Sorry for misunderstanding, next time I will scroll entire text. BTW, would you say 'press ENTER key' or 'press the ENTER key'?

Reply:
'press **the** ENTER key' because _ENTER key_ is not a proper noun.